hopsworks-api/src/main/java/io/hops/hopsworks/api/jupyter/JupyterService.java

```diff
@@ -437,6 +475,32 @@ private void stop(String hdfsUser) throws AppException {
         dfsService.closeDfsClient(dfso);
       }
     }
+
+    try {
+      String experimentsIndex = this.project.getName() + "_experiments";
```

Comment:
@robzor92 `_experiments` should be declared as a constant [here](https://github.com/logicalclocks/hopsworks/blob/master/hopsworks-common/src/main/java/io/hops/hopsworks/common/util/Settings.java#L1703)

Reply:
Will do.